lms/djangoapps/lms_xblock/runtime.py

```diff
@@ -29,7 +29,7 @@ def handler_url(block, handler_name, suffix='', query='', thirdparty=False):
         # Be sure this is really a handler.
         #
         # We're checking the .__class__ instead of the block itself to avoid
-        # auto-proxying from Descriptor -> Module, in case descriptors want
+        # auto-proxying from block -> Module, in case blocks want
```

Comment:
```suggestion
        # auto-proxying from Descriptor -> Module, in case blocks want
```

Let's keep it like this to be more accurate. This part might not be needed anymore, but removing it is not a part of this PR.

Reply:
reverted